src/test/ui/const-generics/min_const_generics/const-expression-suggest-missing-braces.rs

```diff
@@ -0,0 +1,55 @@
+#![feature(min_const_generics)]
```

Comment:
I would like to see how this interacts now with the "invalid turbofish" recovery (just removing `::` from all cases) to make sure it doesn't break down.

Reply:
Good point: I'll add some test cases.

Reply:
At the moment turbofish takes priority and it simply suggests `::` be added. So it would take two attempts for a user to fix both problems, but I think that's okay.

Reply:
That's good to know.